eval_intern.h

```diff
@@ -300,6 +300,8 @@ void rb_vm_set_progname(VALUE filename);
 VALUE rb_vm_cbase(void);
 
 /* vm_backtrace.c */
+#define BACKTRACE_START 0
+#define ALL_BACKTRACE_LINES -1
```

Comment:
I think if we are going to make these constants public API, they should start with `RUBY_` or `RB_.`